web/portal/client/src/entities/HolidayDate/Action/ImportHolidayDates.tsx

```diff
@@ -0,0 +1,108 @@
+import { DropdownChoices } from '@irontec/ivoz-ui';
+import { StyledTable } from '@irontec/ivoz-ui/components/List/Content/Table/ContentTable.styles';
+import { StyledDropdown } from '@irontec/ivoz-ui/services/form/Field/Dropdown/Dropdown.styles';
+import _ from '@irontec/ivoz-ui/services/translations/translate';
+import {
+  Checkbox,
+  SelectChangeEvent,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+} from '@mui/material';
+
+type ImportHolidayDatesMappingTablePropsType = {
+  csv: Array<string>[];
+  columns: string[];
+  setColumns: (value: string[]) => void;
+  ignoreFirstLine: boolean;
+  setIgnoreFirstLine: (value: boolean) => void;
+};
+
+const ImportHolidayDatesMappingTable = (
+  props: ImportHolidayDatesMappingTablePropsType
+): JSX.Element => {
+  const { columns, setColumns, csv, ignoreFirstLine, setIgnoreFirstLine } =
+    props;
+
+  const choices: DropdownChoices = [
+    { id: 'name', label: 'Name' },
+    { id: 'eventDate', label: 'Event data' },
+  ];
+
+  const changeColumnMappingHandler = (event: SelectChangeEvent) => {
+    const { name, value } = event.target;
+    const newVal = [...columns];
+    newVal[parseInt(name, 10)] = value;
+    setColumns(newVal);
+  };
+
+  const changeIgnoreFirstLineHandler = (
+    event: React.ChangeEvent<HTMLInputElement>,
```

Comment:
Variable not used, please remove.

Reply:
https://github.com/irontec/ivozprovider/blob/fec020256e746d8672e2ffdbf7f468bce8337c07/web/portal/client/src/entities/HolidayDate/Action/ImportHolidayDates.tsx#L100

Am I missing something?

Reply:
If you are not using the variable but is needed for any other reason you can rewrite it like "_"